src/Extensions/azure-storage/azure-storage/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor.Egress;
+using Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob;
+using System.Text.Json;
+
+namespace Microsoft.Diagnostics.Monitoring.AzureStorage
+{
+    internal class Program
+    {
+        private static Stream StdInStream = null;
+        static async Task Main(string[] args)
+        {
+            string jsonConfig = Console.ReadLine();
```

Comment:
I think this should be a command line parameter as a file path:
1. Avoids mixed mode content on the stdin stream.
2. Makes it way easier to statically create a file for automated testing and debuggability purposes.